tests/publiccloud/prepare_tools.pm

```diff
@@ -54,10 +54,13 @@ sub run {
         }
         assert_script_run('aws --version');
     }
-    zypper_call('-q in python-ec2uploadimg');
-    assert_script_run("curl " . data_url('publiccloud/ec2utils.conf') . " -o /root/.ec2utils.conf");
     record_info('EC2', script_output('aws --version'));
```

Comment:
don't you want to move this line below your changes ?

Reply:
I think the if/else block right before belongs to installing aws-cli. Thats why I changed the order and gave the ec2imgutils a new block.